### PR TITLE
add StreamGroupInfo EntriesRead and Lag

### LINKED ISSUE #2467: Support for entries-read and lag properties of XINFO GROUPS result in Redis 7
In Redis 7 command XINFO GROUPS returns two new properties for each consumer group: entries-read and lag.

Are there any plans for supporting them in StreamGroupInfo struct?

https://redis.io/commands/xinfo-groups/

### PR DESCRIPTION
**Pull request by tvdias**
Fixes #2467

### CODE REVIEW

File: src/StackExchange.Redis/APITypes/StreamGroupInfo.cs [hunk 1]
```diff
@@ -33,4 +35,14 @@ internal StreamGroupInfo(string name, int consumerCount, int pendingMessageCount
     /// The Id of the last message delivered to the group.
     /// </summary>
     public string? LastDeliveredId { get; }
+
+    /// <summary>
+    /// Total number of entries the group had read.
+    /// </summary>
+    public int? EntriesRead { get; }
```
Comment (mgravell) [inline src/StackExchange.Redis/APITypes/StreamGroupInfo.cs:42]: Should this be int64?

**Comment by mgravell** [comment]
Suggest also:

- release notes update
- test if possible (if server version is blockage, fine)

File: src/StackExchange.Redis/APITypes/StreamGroupInfo.cs (hunk 1 above)
Comment (tvdias) [reply src/StackExchange.Redis/APITypes/StreamGroupInfo.cs]: Updated. Does it make sense to update `lag` for the same reason? Although that should unusual to have a consumer group being created and reading messages from the beggining on very large streams.

Comment (tvdias) [reply src/StackExchange.Redis/APITypes/StreamGroupInfo.cs]: Updated. Does it make sense to update `lag` for the same reason? Although that should be unusual to have a consumer group being created and reading messages from the beginning on very large streams.

Comment (mgravell) [reply src/StackExchange.Redis/APITypes/StreamGroupInfo.cs]: it sounds *unusual*, but if it is theoretically possible, we should probably make the `Lag` 64-bit too, I guess.

Comment (tvdias) [reply src/StackExchange.Redis/APITypes/StreamGroupInfo.cs]: Fixed on c57fbfbd129a9b4aca88b81e4d44a6b4e329739d and c57fbfbd129a9b4aca88b81e4d44a6b4e329739d.

**Comment by tvdias** [comment]
> * release notes update
Fixed on 287bacd6c4380a63c75a7b1a0f268e92933b6fde

> * test if possible (if server version is blockage, fine)
Fixed on f0f0a1d1c460fb3152f97d71794996e2f6391e4c

**Comment by tvdias** [comment]
Hello @mgravell  , can you please take a new look?

And please use the `squash it on merge` option, as I had created some dummy commits to trigger the builds (idk if I could manually do it).

**Review by NickCraver — APPROVED** [review]
Current is looking great - thank you!

**Pull request merged**